subtest/docs/documentation.md

```diff
@@ -0,0 +1,174 @@
+# Distributed learning contributivity
+# Work in progress
+
+## Summary
+
+- Quick start
+- Results
+- Contributivity mesuring methods
+
+## Prerequisities
+
+At root folder:
+- `pip install -r requirements.txt`
+- `pip install -i https://test.pypi.org/simple/ subtest==0.0.0.6`
+
+Note: This is the temporary package for our library.
+
+## Check out our tutorials!
+
+For a better start and a quick understanding of how our library work, we recommand to take a look at our Notebook based tutorials.
+
+## Quick start
+
+### Important classes:
+
+#### Dataset
+
+This is where you define your dataset and relatives objects such as preprocessing functions and model generator.
+
+##### Dataset
+
+This is the structure of the dataset generator:
+
+```python
+dataset = dataset.Dataset(
+    "name",
+    X_train,
+    X_test,
+    y_train,
+    y_test,
+    input_shape,
+    num_classes,
+    preprocess_dataset_labels,      # See below
+    generate_new_model_for_dataset  # See below
+)
+```
```

Comment:
We have a risk this might be a bit unclear for a newcomer: maybe explain that adding a dataset means creating a dataset module (like for already existing ones), and that the dataset constructor feeds from it

Reply:
I do not think that using a new dataset requieres a new module
1- if the package is installed via pip, finding the folder containing the other dataset's modules is not trivial. 
2- A good way to add a new dataset is presented in the example notebooks, in an object oriented way